train.py

```diff
@@ -478,6 +478,9 @@ def main(args):
 		epoch, iteration = checkpoint['epoch'], checkpoint['iteration']
 		if args.train_data_path == checkpoint['args']['train_data_path']:
 			sampler.load_state_dict(checkpoint['sampler_state_dict'])
+			if args.iterations_per_epoch and iteration and iteration % args.iterations_per_epoch == 0:
+				sampler.batch_idx = 0
+				epoch += 1
```

Comment:
bug fix. this if clause is needed, if you continue the training, starting from some checkpoint, and want a predictable continuation of the epoch count and the iterations count.